web/frontend/src/components/slam/2d-renderer.vue

```diff
@@ -176,21 +180,8 @@ const createAxisHelper = (name: string, rotation: number): THREE.AxesHelper => {
 };
 
 // create the background gray grid
-const createGridHelper = (boundingBox: THREE.Box3): THREE.GridHelper => {
-  const deltaX = Math.abs(boundingBox.max.x - boundingBox.min.x);
-  const deltaZ = Math.abs(boundingBox.max.z - boundingBox.min.z);
-  let maxDelta = Math.round(Math.max(deltaX, deltaZ) * gridHelperScalar);
-
-  /*
-   * if maxDelta is an odd number the x y axes will not be layered neatly over the grey grid
-   * because we round maxDelta and then potentially decrease the value, the 1 meter grid spacing
-   * is bound to have a margin of error
-   */
-  if (maxDelta % 2 !== 0) {
-    maxDelta -= 1;
-  }
-
-  const gridHelper = new THREE.GridHelper(maxDelta, maxDelta, backgroundGridColor, backgroundGridColor);
+const createGridHelper = (): GridHelper => {
```

Comment:
This function is greatly simplified, instead of attempting to generate a grid based on PCD size, we just create an enormous grid because there's no longer a performance penalty for size. The previous THREE.js grid implementation needs to create geometry for every single line on the CPU. This new grid just creates a plane and draws lines on the GPU.

Reply:
Oh interesting, I missed that `maxDelta` was being used for not only number of divisions but also cell size. I'm going to slack a few questions to you to clear this up.

Reply:
Follow up after talking with Nick: the point clouds returned from the server have positions that are in SI units. Therefore all we need to do is set the grid cell size to a static value of `1` to ensure a size of 1m, because we also operate under the idea that the three.js world is in SI units.